preseed/provision.erb

```diff
@@ -141,7 +141,8 @@ d-i grub-installer/only_debian boolean true
 d-i grub-installer/with_other_os boolean true
 <% if @host.params['install-disk'] -%>
 d-i grub-installer/bootdev string <%= @host.params['install-disk'] %>
-<% else -%>
+<% end -%>
+<% if @host.operatingsystem.name == 'Debian' and @host.operatingsystem.major.to_i == 8 -%>
```

Comment:
we can do that, but the thought was, that with Ubuntu/vivid the topic will bubble up again.